### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by xiaoliwang-717** [SubmissionsIn/MFLVC#5]
There might be an error in the 85-th line on metric.py, it should include in the 'for'  loop.

**Comment by SubmissionsIn** [comment]
Line-72 and -85 are used to compute the clustering predictions of all views as Eq. (12).

**Comment by xiaoliwang-717** [comment]
Thanks for your reply, I mean line-85 should be included in the 'for step, (xs, y, _) in enumerate(loader):'， if not, the size of the labels_vector will be the same as the batch_size. I found this issue when I run the train.py, maybe you can test it.

**Comment by xiaoliwang-717** [comment]
Thank you for your reply. I have checked it carefully and found that there may be something wrong with the code I downloaded. Causes' soft_vector.extend(q.cpo ().detach().numpy()) 'outside' for step, (xs, y, _) in enumerate(loader): '.

**Comment by SubmissionsIn** [comment]
Yes. That's exactly what Line-81 does.
